app/services/submit_to_app_store/prior_authority/quote_payload_builder.rb

```diff
@@ -3,20 +3,25 @@ module PriorAuthority
     class QuotePayloadBuilder
       def initialize(application)
         @application = application
+        @primary_quote = application.primary_quote
       end
 
       def payload
         @application.quotes.map do |quote|
-          form = ::PriorAuthority::Steps::ServiceCostForm.build(quote, application: @application)
-
           quote.as_json(only: ATTRIBUTES).merge(
-            cost_type: form.cost_type,
-            item_type: form.item_type,
             document: document(quote)
-          )
+          ).merge(primary_quote_attributes)
         end
       end
 
+      def primary_quote_attributes
+        form = ::PriorAuthority::Steps::ServiceCostForm.build(@primary_quote, application: @application)
```

Comment:
It might make sense to memoize this method, otherwise you're having to recalculate the cost type and item type for every quote.